packages/itwin/geo-tools/package.json

```diff
@@ -88,31 +89,32 @@
     "mocha": "^10.2.0",
     "nyc": "^15.1.0",
     "raf": "^3.4.0",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-redux": "^7.2.6",
+    "react": "^18.3.1",
+    "react-dom": "^18.3.1",
+    "react-redux": "^7.2.9",
     "react-test-renderer": "^17.0.2",
-    "redux": "^4.1.0",
+    "redux": "^4.2.1",
     "rimraf": "^3.0.2",
     "sinon": "^17.0.1",
     "sinon-chai": "^3.7.0",
     "source-map-support": "^0.5.6",
     "ts-node": "^7.0.1",
     "ts-sinon": "^2.0.1",
-    "typescript": "~4.6.0",
+    "typescript": "~5.7.2",
     "xmlhttprequest": "^1.8.0"
   },
   "peerDependencies": {
-    "@itwin/appui-abstract": "^4.0.0",
-    "@itwin/appui-react": "^4.10.0",
-    "@itwin/components-react": "^4.10.0",
-    "@itwin/core-bentley": "^4.0.0",
-    "@itwin/core-common": "^4.0.0",
-    "@itwin/core-frontend": "^4.0.0",
-    "@itwin/core-geometry": "^4.0.0",
-    "@itwin/core-react": "^4.10.0",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
+    "@itwin/appui-abstract": "^4.10.7",
+    "@itwin/appui-react": "^5.2.0",
```

Comment:
we should keep the peers at the lowest supported version, aka `^5.0.0` for app-ui

Reply:
Done